src/app/annotationLibrary/annotationItem.js

```diff
@@ -0,0 +1,21 @@
+angular.module("bawApp.components.annotationItem", [])
+    .component("annotationItem", {
+        templateUrl: "annotationLibrary/annotationItem.tpl.html",
+        controller: [
+            "$scope",
+            "$http",
+            function ($scope, $http) {
+
+                this.audioElement = {
```

Comment:
Annotation item now uses ng-audio rather than just plain old audio element, and the model containing defaults is based on how it already works in the listen page, where this object is initialised in the listen controller in a similar way. 
I have removed the muted and autoplay values, but ng-audio complains when it doesn't get initalised values for position and volume.